samples/apps/csv-data-chat-webapp-streamlit/app.py

```diff
@@ -0,0 +1,161 @@
+import streamlit as st
+import pandas as pd
+import numpy as np
+import matplotlib.pyplot as plt
+import semantic_kernel as sk
+import re
+import traceback
+import asyncio
+
+from semantic_kernel.connectors.ai.open_ai import AzureChatCompletion
+
+from io import StringIO
+
+async def main():
```

Comment:
We've restructed the samples folder into the following subfolders. Could you please move this sample to either the `demos` directory or a new `concept` sub-directory? If moving to the concept directory, please also update the README.md.

![image](https://github.com/microsoft/semantic-kernel/assets/35585003/ee1769a1-9634-4aab-9142-6904e1fb5601)